functors task (1)/student/include/impl.h

```diff
@@ -8,6 +8,24 @@
  */
 struct BeerOrganizer
 {
+    size_t firstBeer = static_cast<int>(BeerBrand::HoeGaarden);
```

Comment:
prefer brace initialization
you forgot about static_cast<int> - you can avoid such kind of errors with "{}" init